README.md

```diff
@@ -1,2 +1,75 @@
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+  
+
 # Leaderboard
- The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
+
+The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
+All data is preserved thanks to the external. [Leaderboard API service.](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3)
+
+  
+
+[live version](https://elasfarc.github.io/Leaderboard/dist/)
+
+  
+
+![screen](https://raw.githubusercontent.com/elasfarc/Leaderboard/styling/ss3.png)
+
+  
+
+## Built With
+
+  
+
+- Javascript
+
+- CSS
+
+- Html
+
+  
+
+## Getting Started
+
+  
+
+**To get the local copy of the project up and running you will need to run the following commands on your terminal:**
+
+  
+
+`git clone https://github.com/elasfarc/To-Do-list.git`
+
+  
+
+To install all the available dependecies run:
+
+`npm install`
+
+  
```

Comment:
- [ ] Great job explaining how to run the app locally in such a friendly way. However, I'm afraid you might have missed the `npm run webpack` CLI for building the code files to the _dist_ directory. 🙏